### DISCUSSION TITLE
Indenting

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#4097] [Q&A] [answered]
How can I control the indentation of paragraphs for an entire document?  I see that there is a yaml header option `indenting: ` but the documentation is not very clear about how to use it. 

In R Markdown, I would use: 

```
header-includes:
   - \usepackage{indentfirst}
   - \setlength{\parindent}{4em}
```

Is the `indenting:` argument in Quarto supposed to supplant that somehow?

**Comment by cscheid** [comment]
You can include latex snippets in quarto as well: https://quarto.org/docs/output-formats/pdf-basics.html#latex-includes

**Reply by eteitelbaum** [reply]
Yes I understood that part, but I am wondering if there is a way to do it with a YAML argument. `indenting:` sounds promising but I could not figure out how to use it.

**Reply by cscheid** [reply]
The `indenting` key is only supported in `context` formats, so I think it's unlikely to be the one you need.

**Reply by eteitelbaum** [reply]
Just following up on this, if there is a YAML argument for line height (e.g. `linestretch`) then it would also make sense to have something for paragraph indents. The reason is that once you increase the line height, you immediately lose the ability to differentiate between paras.

**Comment by cderv** [comment, marked as answer]
Pandoc defines the variables for the template and we use them:
https://pandoc.org/MANUAL.html#variables-for-latex

It seems there is a `indent` parameter that can be set to `true` to avoid default Pandoc adjustment 
https://github.com/jgm/pandoc/blob/main/data/templates/default.latex#L204-L216
and we use the same 
https://github.com/quarto-dev/quarto-cli/blob/91fd1456dc969bb5de9f4f1c15ff8455283715cf/src/resources/formats/pdf/pandoc/template.tex#L179-L191

Pandoc's doc says (https://pandoc.org/MANUAL.html#layout)

> indent
> if true, pandoc will use document class settings for indentation (the default LaTeX template otherwise removes indentation and adds space between paragraphs)

If you think the default template could benefit more feature, I would open a feature request in Pandoc directly to make your case.
